src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -121,6 +117,16 @@ export default {
 		currentTabIndex() {
 			return this.tabs.findIndex(tab => tab.id === this.activeTab)
```

Comment:
```suggestion
			return this.sortedTabs.findIndex(tab => tab.id === this.activeTab)
```
The keyboard navigation is currently messed up because of this.

Reply:
It also needs to be adjusted in the `focus*Tab` methods below.